README.md

```diff
@@ -109,9 +109,9 @@ Flags:
       --debuginfo-strip            Only upload information needed for
                                    symbolization. If false the exact binary the
                                    agent sees will be uploaded unmodified.
-      --debuginfo-upload-retry-count=3
-                                   The number of times to retry uploading
-                                   debuginfo files.
+      --debuginfo-upload-max-parallel=25
```

Comment:
Glad this is configurable now! I wonder what the right default value is, perhaps 25 is very high? We've seen debuginfos that weight 100s of GB which could use too much bandwidth perhaps

Reply:
Happy to reduce the default value 💯